mlflow/server/js/src/experiment-tracking/components/CompareRunView.css

```diff
@@ -1,29 +1,65 @@
-.compare-table th.row-header {
-  min-width: 120px;
+.sticky_header {
+  position: sticky;
+  position: -webkit-sticky;
+  left: 0;
+}
+
+.compare-table .table-block {
+  display: block;
+  overflow: auto;
+  scrollbar-width: none;
+}
+
+.no-scrollbar::-webkit-scrollbar {
+  display: none
 }
 
 .compare-table th.inter-title {
   padding: 20px 0 0;
   background: transparent;
+  position: static;
+}
+
+.compare-table .head-value {
+  overflow: hidden;
+  overflow-wrap: break-word;
 }
 
-.compare-table td.data-value,
-.compare-table th.data-value {
+.compare-table .data-value {
   overflow: hidden;
-  max-width: 120px;
   text-overflow: ellipsis;
 }
 
+.hover-text {
+  visibility: hidden;
+  min-width: 50px;
+  max-width: 300px;
+  background-color: yellow;
+  color: black;
+  text-align: center;
+  padding: 5px 0;
+  border-radius: 6px;
+  overflow: hidden;
+  pointer-events: none;
+  position:absolute;
+  z-index: 1;
+}
```

Comment:
Do we really need these styels?

Reply:
Removed this, replace it with antd Tooltip component.